pkg/auth/webapp/context_holder_middleware.go

```diff
@@ -20,8 +22,10 @@ func (m *ContextHolderMiddleware) Handle(next http.Handler) http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 		// This middleware only creates the holder of context.
 		// This enables the holder to be mutated later in other places.
-		var empty web.CSPNonceContextValue
-		ctx := web.WithCSPNonce(r.Context(), &empty)
+		var emptyCSPNonceContext web.CSPNonceContextValue
+		var emptyUIParamContext uiparam.T
```

Comment:
Just curios, why we need to modify something related to ui param and csp nonce in this pr?

Reply:
In v1, we don't need js to make the error page works fine. But in v2, we need js to set the alignment attribute so discovered this bug :sosad: